### DISCUSSION TITLE
Using on Windows

### DISCUSSION THREAD

**Discussion by ITheEqualizer** [celzero/rethink-app#918] [Q&A]
How can I use rethink DNS and its features on Windows?

**Comment by ignoramous** [comment]
Hi there, 

You can't use the RDNS app on Windows. Checkout similar apps like [Glasswire](https://glasswire.com) and [PortMaster](https://github.com/safing/PortMaster).

If you mean, you want to use the resolver, then:

1. Go to [rethinkdns.com/configure](https://rethinkdns.com/configure)
2. Select the blocklists you want enabled.
3. Tap on the underlined-text shown below the search bar, next to the apple 🍎 icon.
   1. Tapping on the text should have copied a DNS-over-HTTPS URL to your clipboard.
   2. You can now paste that DNS-over-HTTPS URL in your browser or OS settings.
4. [Refer this guide](https://archive.is/pDLYm) on how to setup DNS-over-HTTPS on Windows 10.
   1. You can optionally also setup DNS-over-HTTPS [in your favourite browser](https://archive.is/oTP11).

Let me know if anything is not clear or confusing.

**Reply by Saeid008** [reply]
Hi. you app is great.

if you can build a window's RDNS app for Microsoft window and Linux(Ubuntu,...) it will be great then.

Thanks.
